src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/RemoveApplicantDetailsController.java

```diff
@@ -47,8 +73,28 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> removeDetails(
         validateCaseData(callback);
 
         Map<String, Object> caseData = caseDetails.getData();
-        String applicantRepresented = caseData.get(APPLICANT_REPRESENTED).toString();
```

Comment:
toString can lead to NullPointer - check NullPointer

Reply:
Done